OrbitCore/CaptureData.h

```diff
@@ -79,6 +81,11 @@ class CaptureData {
     thread_names_.insert_or_assign(thread_id, std::move(thread_name));
   }
 
+  const FunctionStats& GetOrInsertFunctionStats(uint64_t function_address);
```

Comment:
Maybe this needs to be converted to std::optional and the absence of value handled on the caller side?

Reply:
Not 100%. Thought a lot of this case. This will be called by the UI to query function stats e.g. to display in the live view. If the hooked function was not called in the capture there will be no call update the stats, and thus no entry for this function. 
I see three alternatives to what I've done:
1. Use std::optional and handle the empty case in the client.
2. Change the method to GetOrDefaultFunctionStats and return an empty FunctionStats. This would require to actually return a copy rather then a const ref.
3. In constructor, create empty entries for all selected functions and make this function fail if function is not there (e.g. by using `at`).

Reply:
in case of 2 you do not have to return copy if you create a static const FunctionStats

Reply:
Right. So which do you prefer?

Reply:
As discussed, changed it to 2.